src/app/cli/src/services/config/models.rs

```diff
@@ -309,6 +315,55 @@ impl Default for JupyterConfig {
 
 ////////////////////////////////////////////////////////////////////////////////////////
 
+#[skip_serializing_none]
+#[derive(Debug, Clone, Serialize, Deserialize)]
+#[serde(deny_unknown_fields)]
+#[serde(rename_all = "camelCase")]
+#[serde(tag = "kind")]
+pub enum DatabaseConfig {
+    Sqlite(SqliteDatabaseConfig),
```

Comment:
Why is there no `InMemory` option? I'm assuming all the rest are for testing purposes only and `kamu ui` / `kamu system api-server` will continue to use in-memory storage by default? We won't be having an sqlite db as part of our default workspace, won't we?

Reply:
yes, that's a very relevant comment -- the current configuration assumes that if the database section is not specified, `InMemory` will be used.

It is really worth making this more explicit, while also leaving that the absence of the option will result in `InMemory`.

Reply:
Done